From 5e0c34139bbd577b09522056cbdf4812275eb757 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Sat, 19 Nov 2016 21:30:46 +0200
Subject: [PATCH] Generate man pages with UTF-8 characters

This fixes totally incorrectly rendered manual pages like
perlunicook and perltw.

TODO: should probably be a Configure option for proper upstreaming.

Bug-Debian: https://bugs.debian.org/840211
Patch-Name: debian/installman-utf8.diff

---
 installman | 1 +
 1 file changed, 1 insertion(+)

diff --git a/installman b/installman
index 53117ab..80d9bc4 100755
--- a/installman
+++ b/installman
@@ -153,6 +153,7 @@ sub pod2man {
 	$manpage = "${mandir}/${manpage}.${manext}";
 
         my $parser = Pod::Man->new( section => $manext,
+                                    utf8    => 1,
                                     official=> 1,
                                     center  => 'Perl Programmers Reference Guide'
                                   );
